src/Bitcoin/FeeCalculator.h

```diff
@@ -10,49 +10,63 @@
 
 namespace TW::Bitcoin {
 
+inline constexpr double gDefaultBytesPerInput{148};
+inline constexpr double gDefaultBytesPerOutput{34};
+inline constexpr double gDefaultBytesBase{10};
+inline constexpr double gSegwitBytesPerInput{101.25};
+inline constexpr double gSegwitBytesPerOutput{31};
+inline constexpr double gSegwitBytesBase{gDefaultBytesBase};
 /// Interface for transaction fee calculator.
 class FeeCalculator {
-public:
-    virtual int64_t calculate(int64_t inputs, int64_t outputs, int64_t byteFee) const = 0;
-    virtual int64_t calculateSingleInput(int64_t byteFee) const = 0;
+  public:
```

Comment:
I prefer 4-space indentation, a 2-space indentation confuses many editors, and they revert to 2-space indentation.
Either two full levels for `public` and the members, or `public` w/o indentation

Reply:
Example with IndentAccessModifiers :

https://user-images.githubusercontent.com/21139416/171010250-12d2060c-ffb8-4acf-b46c-e6c2574e739c.mov

Reply:
> I prefer 4-space indentation, a 2-space indentation confuses many editors, and they revert to 2-space indentation. Either two full levels for `public` and the members, or `public` w/o indentation

Should be fixed now, I've modified the `.clang-format` to respect this specific behaviour, please let me know if that's better now